tools/cold-backup/README.md

```diff
@@ -0,0 +1,85 @@
+# Strimzi backup
+Bash script for cold/offline backups of *Strimzi* clusters on *Kubernetes/OpenShift*.
+
+If you think you do not need a backup strategy for Kafka as it has embedded data replication,
+then try to immagine a misconfiguration/bug/security-breach deleting all your data. For hot/online
+backups, you should look at *MirrorMaker2* to sync with a remote cluster, but this comes with
+additional complexities and required resources.
+
+To run the script you must be logged in as *cluster-admin* user. The script will stop the operator
```

Comment:
Does it really need to be _cluster-admin_ user? I don't see anything what would require it. It might be better to list the permissions the user needs (not necessarily listing every RBAC, but for example _you need to be abel to work with PVC and use the Strimzi custom resources_ etc.)

Reply:
Right. Let me double check this.